packages/calcite-components/src/components/list-item-group/list-item-group.tsx

```diff
@@ -49,7 +49,7 @@ export class ListItemGroup implements InteractiveComponent {
    * Fires when changes occur in the default slot, notifying parent lists of the changes.
    */
   @Event({ cancelable: false })
-  calciteInternalListItemGroupDefaultSlotChange: EventEmitter<DragEvent>;
+  calciteInternalListItemGroupDefaultSlotChange: EventEmitter<void>;
```

Comment:
it didn't actually emit this type, and no one was listening for it to emit this type